README.md

```diff
@@ -185,3 +185,225 @@ Open VirtualBox
 File -> Host Network Manager -> Create
 This should create a hostadapter named vboxnet0
 ```
+Tutorial: install and configure NixOS for nix-bitcoin on your own hardware
+---
+## 1. NixOS installation
+
+This is borrowed from the [NixOS manual](https://nixos.org/nixos/manual/index.html#ch-installation). Look there for more information.
+
+1. Obtain latest NixOS. For example:
```

Comment:
also verify sigs?

Reply:
Argh... looks like NixOS releases are not signed ... *sigh*. There are a few discussions around it, but nix people there don't understand the threat model.